version/version.go

```diff
@@ -0,0 +1,47 @@
+// Package version records the verion of the ts-bridge project.
+// It also includes helper methods to return a user-agent that can be used in requests.
+package version
+
+import (
+	"fmt"
+	"runtime"
+
+	"google.golang.org/api/googleapi"
+)
+
+const (
+	// Project is the name of this project.
+	Project = "ts-bridge"
+
+	// Ver is the current version of this project.
+	Ver = "0.1"
```

Comment:
We need to adhere to [semantic versioning](https://semver.org/), so this needs to be "0.1.0"